doc/api/modules.md

```diff
@@ -180,9 +180,12 @@ regarding which files are parsed as ECMAScript modules.
 If `--experimental-require-module` is enabled, and the ECMAScript module being
 loaded by `require()` meets the following requirements:
 
-* Explicitly marked as an ES module with a `"type": "module"` field in
-  the closest package.json or a `.mjs` extension.
-* Fully synchronous (contains no top-level `await`).
+* One of these conditions are met:
+  1. The file has a `.mjs` extension.
+  2. The file has a `.js` extension, and the closest `package.json` contains `"type": "module"`
+  3. The file has a `.js` extension, the closest `package.json` does not contain
+     `"type": "commonjs"`, and `--experimental-detect-module` is enabled.
+* The module is fully synchronous (contains no top-level `await`).
```

Comment:
I think it’s easier to understand that both bullets must evaluate to true if there’s an “and” connecting them.

```suggestion
* The module is fully synchronous (contains no top-level `await`); and
* One of these conditions are met:
  1. The file has a `.mjs` extension.
  2. The file has a `.js` extension, and the closest `package.json` contains `"type": "module"`
  3. The file has a `.js` extension, the closest `package.json` does not contain
     `"type": "commonjs"`, and `--experimental-detect-module` is enabled.
```